Resources/Private/Templates/Email/Layouts/Default.html

```diff
@@ -0,0 +1,9 @@
+Dear Administrator,
+
+a form has been submitted:
+
+<f:render section="Content" />
+
+--
+FormBuilder
+https://github.com/cron-eu/neos-formbuilder
```

Comment:
funny, but I think this would rather confuse our customer (he might thing this is a spam, linking to a foreign page). No greetings at the end is enough, this is an automated message anyway.

Reply:
The template is only an example template and is not used by the SHOP the message I have changed anyway. :)